radixdlt/src/main/java/com/radixdlt/consensus/safety/SafetyState.java

```diff
@@ -64,17 +78,39 @@ public boolean equals(Object o) {
 			return false;
 		}
 		SafetyState that = (SafetyState) o;
-		return Objects.equals(lastVotedRound, that.lastVotedRound)
-			&& Objects.equals(lockedRound, that.lockedRound);
+		return Objects.equals(lastVotedView, that.lastVotedView)

```

Comment:
Don't see any EqualsVerifier test for equals(...) and hashCode().

Reply:
It's in SafetyStateTest

Reply:
Arrgh.  Missed it 🤦‍♂